packages/plugin-commands-doctor/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@pnpm/plugin-commands-doctor",
+  "version": "1.0.0",
```

Comment:
make it 0.0.0 and create a changeset to bump it to major

Reply:
ok